core/templates/dev/head/player/exploration_player.html

```diff
@@ -33,6 +33,55 @@
 {% endblock navbar_breadcrumb %}
 
 {% block local_top_nav_options %}
+  <div ng-controller="LearnerLocalNav">
```

Comment:
Why two of these (here and in the line below)? Also note the inclusion of learner_local_nav.html on line 85; this should be moved into there.